docs/dev_guide/contents/newcomers.rst

```diff
@@ -191,6 +191,25 @@ If you get stuck or want help, just `ask here`_!
 .. _sunpy repository: https://github.com/sunpy/sunpy
 .. _ask here: https://openastronomy.element.io/#/room/#sunpy:openastronomy.org
 
+Setting up a development environment using Vagrant (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you do not want to install Python to your computer or use a system specific solution, you can use `vagrant`_.
+
+You will need to install `virtualbox`_ and `vagrant`_.
+Then run the following replacing ``<username>`` with your username.
+
+.. code:: bash
+
+    $ git clone https://github.com/<username>/sunpy.git sunpy-git
+    $ cd sunpy-git
+    $ vagrant up && vagrant ssh
+
+This will automatically log you in the VM inside a sunpy-dev conda environment with all dependencies pre-installed.
+
+.. _virtualbox: https://www.virtualbox.org/manual/ch02.html
+.. _vagrant: https://www.vagrantup.com/docs/installation
+
```

Comment:
```suggestion
```

Having had some time to think about this, I changed my mind.
Probably should not advertise this just in case it confuses anyone who is new.